Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /nlisim
```

Comment:
`/opt/nlisim` is a better location for arbitrary software, which follows FHS better: https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
Renamed to /opt/simulation-framework